protobuf/lib/src/protobuf/builder_info.dart

```diff
@@ -128,8 +128,19 @@ class BuilderInfo {
   }
 
   // Map field.
-  void m<K, V>(int tagNumber, String name, int keyFieldType, int valueFieldType,
-      BuilderInfo mapEntryBuilderInfo) {
+  void m<K, V>(int tagNumber, String name, String entryClassName,
+      int keyFieldType, int valueFieldType,
+      [PackageName packageName,
+      CreateBuilderFunc valueCreator,
+      ValueOfFunc valueOf,
+      List<ProtobufEnum> enumValues]) {
+    BuilderInfo mapEntryBuilderInfo = packageName == null
+        ? BuilderInfo(entryClassName)
```

Comment:
Do you need the `?:` if `packageName` is `null` you should just be able to pass it there?

Reply:
Not exactly, since the default is not null. I changed it now, so that `packageName` is the last optional parameter in `m()` and `m()` defines a default value.